### ISSUE TITLE
GTK+ 3: application name odd in window list and about dialog

### ISSUE THREAD

**Issue by az0** [bleachbit/bleachbit#332]
Currently states `Bleachbit.py` instead of BleachBit

Tested with Ubuntu 16.04, GTK 3.18.9, commit ede855c

**Comment by az0** [comment]
I confirmed this is still necessary for branch gtk3 as of commit 3a3d923 . System is Ubuntu 18.10

**Comment by mkhon** [comment]
I can not reproduce this on Debian buster, will try to reproduce on Ubuntu 18.04/18.10

**Comment by mkhon** [comment]
Can not reproduce on Ubuntu 18.04 as well

**Comment by az0** [comment]
Could it be a different window manager? I used Ubuntu and Gnome Classic. Also, I launched from the command line like `python bleachbit.py` without installation.

Gnome Classic Window Manager on Ubutnu 18.10
![Screenshot from 2019-04-15 Gnome Classic on Ubuntu 18 10](https://user-images.githubusercontent.com/933680/56178622-2cb61200-5fc0-11e9-9f3e-e9be79723140.png)

Ubuntu window manager (not Wayland) on Ubuntu 18.10
![Screenshot from 2019-04-15 Ubuntu Window Manager on Ubuntu 18 10](https://user-images.githubusercontent.com/933680/56178624-2cb61200-5fc0-11e9-85e0-c1fcad25418b.png)
